src/senaite/core/content/interpretationtemplates.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE.
+#
+# SENAITE.CORE is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2020 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+from plone.dexterity.content import Container
+from plone.supermodel import model
+
+from senaite.core import PloneMessageFactory as _PMF
+
+from zope import schema
+from zope.interface import implementer
+
+
+class IInterpretationTemplates(model.Schema):
+    """Results Interpretation Templates folder interface
+    """
+    title = schema.TextLine(
+        title=_PMF(u'label_title', default=u'Title'),
+        required=True)
```

Comment:
Shall we better  use the `plone.app.dexterity.behaviors.metadata.IBasic` behavior for those types?

Reply:
I thought about this, but the `IBasic` behavior comes with other fields like description, etc.. and I thought that was better to stick to the minimum requirements. We want small objects, less overhead. What we can do is to create our own behavior and apply this behavior to all content "folders".